INSERT INTO reference (name, display_name, path, genome_count, arm_selection, description, example) VALUES ( 'svevo-2','Durum wheat genome (cv. Svevo)','/home/polymarker/data/genomes/160802_Svevo_v2_pseudomolecules/160802_Svevo_v2_pseudomolecules.fasta', '2', 'nrgene', 'An international consortium has generated a high quality reference sequence of the modern durum wheat cultivar Svevo. Whole genome libraries were sequenced with Illumina short paired-end (2 x 250 bp) and long mate pair (up to 8-10 kb) protocols and the reads were assembly with the NR-Gene DeNovoMAGICTM pipeline. In total, a 270x coverage was obtained, and the reads were assembled into 10.5 Gb of sequence with an L50 and L90 length of 6 and 1 Mb, respectively (N50: 493; N90: 2019). The assembly scaffolds have been anchored to a high-density genetic map based on Svevo × Zavitan RIL population and merged into super-scaffolds using Hi-C data; 95% of the scaffolds have been anchored along the chromosomes and 90% oriented. Reference sequence available in [InterOmics](https://www.interomics.eu/durum-wheat-genome)','Kronos4607.chr2A.256748,2A,agaagcttattatgcatatgtggaattcatttattcggaaacaaaggtatgcgtgcaaaatcaagtgttttacggacggtactcgctgaaccctacatgc[C/T]tcgtgtcttgctgaaagagtatgataattcattcacagggtgctagctgatgggcatataccttgggcctgtgagggcttctcccggcttcatggaccgc\nKronos3485.chr3A.750506662,3A,gcttgtttcatgaaaatcatgtttactagagcttagtaattaatagtcttaacatgttaaatagccatcgatgacccaattactggtgtcagaggaccaa[G/A]atgccataatttcttacaaacaagttgtgatgcttcattagttaccactataccaactaagattccttaggagtttgaccatcaaactgtagttttacgg\nKronos3658.chr4A.82557,4A,gacggtgaacttgacctcgggcgcctcggcaacggtggcggcgtaggccttcttgacggcggactcgagggcagggacgaacttgtagcccgcgtatgcg[C/T]cgccagtggtctccttgatggccttgtcgaaggcggactcgaagacggtgaacttgtcgttgacgggggcagcgtcggcggcggtggcggcaatcctgaa\nKronos2065.chr6B.21130,6B,atatgctcaaacatgggtccggtatgctagcccaaagtaatatttttaagcaaatgctgacctcctattgcacgttccagttgctgggcaaacacggttc[C/T]atagtcgcgcttatcataccggaagtagggtgcagccgccttctttgagatcagcccagccagtatgcctacaaacccttccttcctttcctcaggagct');
INSERT INTO reference (name, display_name, path, genome_count, arm_selection, description, example) VALUES ('chinese_spring-1.0','Wheat cv Chinese Spring RefSeq v1.0 (Full sequence)', '/home/polymarker/data/genomes/161010_Chinese_Spring_v1.0_pseudomolecules/161010_Chinese_Spring_v1.0_pseudomolecules.fasta', '3', 'nrgene', 'A whole genome assembly has been carried out by the IWGSC in collaboration with the company NRGene. Using a proprietary algorithm DeNovoMAGIC with Illumina sequencing data a 14.5 Gb assembly was produced.Sequences have been ordered using POPSEQ data and Hi-C (chromosome conformation capture) to generate 21 pseudomolecules representing the majority of the wheat genome. Reference sequence available in [URGI](https://wheat-urgi.versailles.inra.fr/Seq-Repository/Annotations) The assembly is described in [IWGSC (2018)](https://doi.org/10.1126/science.aar7191).', 'Cadenza1697.chr1A.12142209,1A,acaacttttcaaaaaataacacacccacaacctaaacacccttaaatatcaacttctgaggggaggcggcaactagtatcaaaaattcaaaatgtggttc[G/C]cacaactagcgttgccgagacaaaaagaaccagcacttccactgctccatgaattcaagaaacagaccgatggcatgtataagaacagacgtgccga\n BA00591935,3B,gcacatcggagagttcgttggaagagcctagcaccgaggaattccaggaggaagatgtctctgatggagattcagactcgaacgacgagagcaaggggcc[G/A]gaagtaaagcttttcatcagcggagtagttcataataaagaggaggctggagcaaaatcttatgttcgagttcccgctgaaataaataacctggaaaggg\n BA00343846,5A,ccacccctcctcttccccatgcagctacaactgaggcaatgctgccgtctgccttctggagttccccgccttcgatctcgagcacgccatccaggacggc[G/A]ttggtatcgaccgccaacctcaatctggatccggttgtggctaatgtacagactaattcccacacaaaacattcattaagttagcattgtctctttttgg\nBA00122841,7D,ttcccaccacacgcttcagcagttccttggctcgcgatgaccccgactcatcatggcagaactgggtccggaggcccccatcaagacccttgtaaatgta[A/G]aagtacactgccagttgttcgctgccgctgttggtggcatggatgagaaggccaaaggggccaagcgcgccgcgttcagtggcaccaccacttgtactgc');
# INSERT INTO user (username, password)
# VALUES
#   ('test', 'pbkdf2:sha256:50000$TCI4GzcX$0de171a4f4dac32e3364c7ddc7c14f3e2fa61f2d17574483f7ffbb431b4acb2f'),
#   ('other', 'pbkdf2:sha256:50000$kJPKsz6N$d2d4784f1b030a9761f5ccaeeaca413f27f2ecb76d6168407af962ddce849f79');
#
# INSERT INTO post (title, body, author_id, created)
# VALUES
#   ('test title', 'test' || x'0a' || 'body', 1, '2018-01-01 00:00:00');
# #
#
# INSERT INTO reference (name, path, genome_count, arm_selection, description, example)
# VALUES ('Durum wheat genome (cv. Svevo)','/home/polymarker/data/genomes/160802_Svevo_v2_pseudomolecules/160802_Svevo_v2_pseudomolecules.fasta', '2', 'nrgene', 'An international consortium has generated a high quality reference sequence of the modern durum wheat cultivar Svevo. Whole genome libraries were sequenced with Illumina short paired-end (2 x 250 bp) and long mate pair (up to 8-10 kb) protocols and the reads were assembly with the NR-Gene DeNovoMAGICTM pipeline. In total, a 270x coverage was obtained, and the reads were assembled into 10.5 Gb of sequence with an L50 and L90 length of 6 and 1 Mb, respectively (N50: 493; N90: 2019). The assembly scaffolds have been anchored to a high-density genetic map based on Svevo × Zavitan RIL population and merged into super-scaffolds using Hi-C data; 95% of the scaffolds have been anchored along the chromosomes and 90% oriented. Reference sequence available in [InterOmics](https://www.interomics.eu/durum-wheat-genome)','Kronos4607.chr2A.256748,2A,agaagcttattatgcatatgtggaattcatttattcggaaacaaaggtatgcgtgcaaaatcaagtgttttacggacggtactcgctgaaccctacatgc[C/T]tcgtgtcttgctgaaagagtatgataattcattcacagggtgctagctgatgggcatataccttgggcctgtgagggcttctcccggcttcatggaccgc\nKronos3485.chr3A.750506662,3A,gcttgtttcatgaaaatcatgtttactagagcttagtaattaatagtcttaacatgttaaatagccatcgatgacccaattactggtgtcagaggaccaa[G/A]atgccataatttcttacaaacaagttgtgatgcttcattagttaccactataccaactaagattccttaggagtttgaccatcaaactgtagttttacgg\nKronos3658.chr4A.82557,4A,gacggtgaacttgacctcgggcgcctcggcaacggtggcggcgtaggccttcttgacggcggactcgagggcagggacgaacttgtagcccgcgtatgcg[C/T]cgccagtggtctccttgatggccttgtcgaaggcggactcgaagacggtgaacttgtcgttgacgggggcagcgtcggcggcggtggcggcaatcctgaa\nKronos2065.chr6B.21130,6B,atatgctcaaacatgggtccggtatgctagcccaaagtaatatttttaagcaaatgctgacctcctattgcacgttccagttgctgggcaaacacggttc[C/T]atagtcgcgcttatcataccggaagtagggtgcagccgccttctttgagatcagcccagccagtatgcctacaaacccttccttcctttcctcaggagct');
#
#
# INSERT INTO reference (name, path, genome_count, arm_selection, description, example)
# VALUES ('Wheat cv Chinese Spring RefSeq v1.0 (Full sequence)', '/home/polymarker/data/genomes/161010_Chinese_Spring_v1.0_pseudomolecules/161010_Chinese_Spring_v1.0_pseudomolecules.fasta', '3', 'nrgene', ' A whole genome assembly has been carried out by the IWGSC in collaboration with the company NRGene. Using a proprietary algorithm DeNovoMAGIC with Illumina sequencing data a 14.5 Gb assembly was produced.Sequences have been ordered using POPSEQ data and Hi-C (chromosome conformation capture) to generate 21 pseudomolecules representing the majority of the wheat genome. Reference sequence available in [URGI](https://wheat-urgi.versailles.inra.fr/Seq-Repository/Annotations) The assembly is described in [IWGSC (2018)](https://doi.org/10.1126/science.aar7191).', '    Cadenza1697.chr1A.12142209,1A,acaacttttcaaaaaataacacacccacaacctaaacacccttaaatatcaacttctgaggggaggcggcaactagtatcaaaaattcaaaatgtggttc[G/C]cacaactagcgttgccgagacaaaaagaaccagcacttccactgctccatgaattcaagaaacagaccgatggcatgtataagaacagacgtgccga\n BA00591935,3B,gcacatcggagagttcgttggaagagcctagcaccgaggaattccaggaggaagatgtctctgatggagattcagactcgaacgacgagagcaaggggcc[G/A]gaagtaaagcttttcatcagcggagtagttcataataaagaggaggctggagcaaaatcttatgttcgagttcccgctgaaataaataacctggaaaggg\n BA00343846,5A,ccacccctcctcttccccatgcagctacaactgaggcaatgctgccgtctgccttctggagttccccgccttcgatctcgagcacgccatccaggacggc[G/A]ttggtatcgaccgccaacctcaatctggatccggttgtggctaatgtacagactaattcccacacaaaacattcattaagttagcattgtctctttttgg\nBA00122841,7D,ttcccaccacacgcttcagcagttccttggctcgcgatgaccccgactcatcatggcagaactgggtccggaggcccccatcaagacccttgtaaatgta[A/G]aagtacactgccagttgttcgctgccgctgttggtggcatggatgagaaggccaaaggggccaagcgcgccgcgttcagtggcaccaccacttgtactgc');
